### ISSUE TITLE
Build on master / Java 11 fails due test fail in testAllRegressionFilesJavac

### ISSUE THREAD

**Issue by iloveeclipse** [spotbugs/spotbugs#996]
```
 Task :spotbugs-tests:test

edu.umd.cs.findbugs.DetectorsTest > testAllRegressionFilesJavac FAILED
    java.lang.AssertionError: Unexpected bugs (3):
    unexpected RCN At Bug1169.java:[line 69]
    unexpected RCN At Bug1169.java:[line 47]
    unexpected RCN At Bug1169.java:[line 32]
        at org.junit.Assert.fail(Assert.java:88)
        at edu.umd.cs.findbugs.DetectorsTest.checkForUnexpectedBugs(DetectorsTest.java:142)

57 tests completed, 1 failed, 2 skipped

> Task :spotbugs-tests:test FAILED

```

**Comment by iloveeclipse** [comment]
Same on the travis: https://travis-ci.org/spotbugs/spotbugs/jobs/564310590.

So it is Java 11 issue with the test.

**Comment by KengoTODA** [comment]
fixed by #1248
